Makefile

```diff
@@ -1,12 +1,16 @@
 main:
-	g++ -std=c++1y proggy.cpp -o prog -Iinclude
+#	g++ -std=c++1y proggy.cpp -o prog -Iinclude
+	g++ -std=c++14 proggy.cpp -o prog -Iinclude
+	#KOOS:
+	$(info **** c++1y was the draft standard.. use C++14 **** )
 
 #test:
 #	g++ -std=c++1y testy.cpp -o test -Iinclude
 #	g++ -std=c++1y proggy.cpp -o prog
 
 clean:
 	rm prog
-	rm test
-	rm main.d
+#	rm test
+#	rm main.d
+#KOOS: test and main.d does not exist.. (at least on linux)
```

Comment:
fair -- those are artifacts from when I was tempted to add all the tests I had and make shared objects